README.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# soap: A Store of Outbreak Analytics Pipelines
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/epiverse-trace/data_pipelines/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/epiverse-trace/data_pipelines/actions/workflows/R-CMD-check.yaml)
```

Comment:
Minor optional suggestion: add repo status badge, licence badge, and test coverage bade.